Assets/Content/Systems/Interactions/OpenInteraction.cs

```diff
@@ -11,6 +11,7 @@ public class OpenInteraction : IInteraction
         public Sprite icon;
 
         public event EventHandler<bool> OpenStateChange;
+        public event EventHandler<OpenInteractionEventArgs> OpenStateChangeSendEvent;
```

Comment:
Is there a reason for adding a new event instead of modifying the existing one?

Reply:
I was just afraid of messing up something by modifying the existing one so I just did that for now. But it would probably be better to change the existing one, right.